### DISCUSSION TITLE
Pure data wire

### DISCUSSION THREAD

**Discussion by smutax** [shorepine/tulipcc#91] [General]
Hi there. Is it possible to control Tulip/AMY using Pure Data from mobile devices? Usb tethering known to be working on iPhones and Android devices allowing Pure Data to send messages over network protocol. Also the suitability of ESP32-S3-N16R8 variant. Keep up the good work. Many thanks. Cheers

**Comment by bwhitman** [comment]
I think this question may be more suited for the [Alles project](https://github.com/bwhitman/alles). That said, I don't see any reason why you couldn't control Alles (running on Tulip or on its own speaker) over anything that can send UDP.

**Reply by smutax** [reply]
Thanks. Hope you don't mind me mentioning your project to certain @mrfloydst. Would be nice to see a video from him. All the very best.

**Reply by mrfloydst** [reply]
Oh, thanks for the heads up! That looks super interesting. I'd love to make a video on this.
